clang/test/Driver/Xclangas.s

```diff
@@ -0,0 +1,6 @@
+/// Check that -Xclangas/-Xclangas= are passed to -cc1as.
+// RUN: %clang -### -Xclangas -target-feature -Xclangas=+v5t %s 2>&1 | \
```

Comment:
Also add a `-Werror` to test `Arg->claim();`. I haven't tested, but without `Arg->claim();` there will be a warning / or error if -Werror

Reply:
Tested locally and you're right!